gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+##################
+# helper functions
+##################
+is_file_generated(){
+  file_path=$1
+  if [ -e "$file_path" ];then
+    file_name=`basename $file_path`
+    echo "SUCCESS: file $file_name generated"
+  else
+    echo "ERROR: file_path $file_path not generated"
+    exit 1
+  fi
+}
+
+rm_if_exists(){
+  file_path=$1
+  if [ -e "$file_path" ];then
+    echo "WARNING: deleting $file_path"
+    rm $file_path
+  fi
+}
+
+CURR_DIR=`pwd`
+script_real_path=`realpath $0`
```

Comment:
See the perl way in snappy-start-all.sh and other scripts. That is more reliable way that takes care of symlinks etc because realpath is not installed in many default installs.